packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/networks.js

```diff
@@ -33,6 +33,8 @@ export const networks = Object.freeze({
       WebSocket: environment.isTest()
         ? 'ws://localhost:21000'
         : 'wss://iohk-mainnet.yoroiwallet.com:443',
+      TokenInfoService:
+        'https://tokens.cardano.org',
```

Comment:
We might need to whitelist this in the manifest

Reply:
@v-almonacid , this is a temporary value for draft only, supposed to be replaced by a `yoroiwallet.com` address once we get it set up